tailwind.config.js

```diff
@@ -0,0 +1,45 @@
+function withOpacityValue(variable) {
+  return ({ opacityValue }) => {
+    if (opacityValue === undefined) {
+      return `rgb(var(${variable}))`;
+    }
+    return `rgb(var(${variable}) / ${opacityValue})`;
+  };
+}
+
+module.exports = {
+  prefix: "tw-",
+  content: ["./src/**/*.{html,ts}"],
+  safelist: [
+    {
+      pattern: /tw-(border|bg)-(primary|secondary)/,
+    },
+    {
+      pattern: /tw-text-(primary|secondary)-btn-text/,
+    },
+    {
+      pattern: /tw-bg-(primary|secondary)/,
+      variants: ["hover"],
+    },
+  ],
```

Comment:
Some components like `button` might want to dynamically generate classes. Since tailwind uses a JIT these won't be picked up automatically. The alternative is to explicitly define them which might be preferred. But I wanted to highlight that we can safelist items using regex as well.